addons/project/models/project_task.py

```diff
@@ -495,6 +495,7 @@ def _compute_dependent_tasks_count(self):
         if tasks_with_dependency:
             group_dependent = self.env['project.task']._read_group([
                 ('depend_on_ids', 'in', tasks_with_dependency.ids),
+                ('state', 'not in', list(CLOSED_STATES)),
```

Comment:
```suggestion
                ('is_closed', '=', False),
```
`is_closed` can be used here. 🙂